README.md

```diff
@@ -14,57 +14,73 @@ Building and running this node allows you to:
-2. Download the genesis json [lptestnet.json](http://eth-testnet.livepeer.org/lptestnet.json) (depending on your geth version, it may complain about 'genesis.number'. If encountered, delete the "number" field in the json.)
-3. Run `geth init` to initialize your local geth node with testnet genesis block.  For example: `geth --datadir ~/.lpGeth init lptestnet.json`
-4. Start `geth` with the network id `858585` and the Livepeer testnet bootnode. For example: `geth --datadir ~/.lpGeth --networkid 858585 --bootnodes "enode://080ebca2373d15762c29ca8d85ddc848f10a7ffc745f7110cacba4694728325d645292cb512d7168323bd0af1650fca825ff54c8dba20aec8878498fae3ff3c6@18.221.67.74:30303"`
 
-Now the geth node should be running, and it should soon start downloading blocks.
+1. Create a geth data directory. For example: `mkdir ~/.lpGeth`. 
+  * We recommend creating a new directory even if you already have one, so the Livepeer testing data will be stored separately.
+2. Download the genesis json [lptestnet.json](http://eth-testnet.livepeer.org/lptestnet.json)
+  * It can be saved anywhere. It'll just be used once for the next step
+3. Initialize your local geth node with testnet genesis block.  For example: `geth --datadir ~/.lpGeth init lptestnet.json`
+  * Depending on your geth version, you may see a complaint about 'genesis.number' related to your .json file. To fix the issue, delete the "number" field in the json.
+4. Start `geth` with the network id `858585` and the Livepeer testnet bootnode. For example: `geth --datadir ~/.lpGeth --networkid 858585 --bootnodes "enode://080ebca2373d15762c29ca8d85ddc848f10a7ffc745f7110cacba4694728325d645292cb512d7168323bd0af1650fca825ff54c8dba20aec8878498fae3ff3c6@18.221.67.74:30303"`
+  * Now the geth node should be running, and it should soon start downloading blocks.
 
-## Usage
+## Running Livepeer
 
 ### Quick start
-- Make sure you have done all the setup steps. (Have `ffmpeg` installed, have `geth` running on the Livepeer private testnet)
+- Make sure you have successfully gone through the steps in 'Installing Livepeer' and 'Additional Dependencies'.
 
-- Run `livepeer -testnet` to start Livepeer. Make sure you have the `livepeer` executable.  It can be downloaded from the [releases page](https://github.com/livepeer/go-livepeer/releases) (you should download the packages for your OS - darwin for Macs and linux for linux.  You should also rename them to `livepeer` and `livepeer_cli`, and make sure they have executable permissions).
+- Start `geth ` (see step 4 of 'Livepeer private Ethereum testnet').
 
-- Run `livepeer_cli`. You should see a wizard launch in the command line. Make sure you installed the `livepeer_cli` executable.  It can be downloaded from the [releases page](https://github.com/livepeer/go-livepeer/releases).
+- Run `./livepeer -testnet`.
+  * Take note of the log output for your Eth address.  You should see a line: `'Using Eth account: ...'`
```

Comment:
The ETH address is also printed out when you run `livepeer_cli`, so perhaps that's easier to find than requesting that they watch log output?

Reply:
@dob good point! pushed an update with instructions directing users to use livepeer_cli to access info